jerry-main/main-unix.c

```diff
@@ -121,6 +121,33 @@ print_help (char *name)
                       name);
 } /* print_help */
 
+static void
+print_unhandled_exception (jerry_value_t error_value)
+{
+  assert (jerry_value_has_error_flag (error_value));
+
+  jerry_value_clear_error_flag (&error_value);
+  jerry_value_t err_str_val = jerry_value_to_string (error_value);
+  jerry_size_t err_str_size = jerry_get_string_size (err_str_val);
+  jerry_char_t err_str_buf[256];
+
+  if (err_str_size >= 255)
```

Comment:
Is `err_str_size` the number of valid characters, i.e., not counting any terminating zeros? (Seems so!) If so, then a 255 char long string is still perfectly valid as the terminating zero will be written to the 256th byte of `err_str_buf` (which is index 255). Thus, this should either be `err_str_size >= 256` or `err_str_size > 255`.

Note: this is contradicting with a previous comment of @zherczeg